tests/components/splunk/test_init.py

```diff
@@ -120,3 +132,54 @@ def test_event_listener(self, mock_requests):
                     headers={'Authorization': 'Splunk secret'},
                     timeout=10, verify=True)
             self.mock_post.reset_mock()
+
+    def _setup_with_filter(self, mock_requests):
```

Comment:
`mock_requests` isn't used in this function. We don't need to pass it here. It's active in the test, that's enough.

Reply:
Thanks.  Missed that.  I originally started trying to model my setup after another used for this component.  I eventually deviated and went a whole other way (taking out use of mock_requests).  Updated.